bedrock/firefox/views.py

```diff
@@ -577,11 +587,24 @@ def get_template_names(self):
             else:
                 template = 'firefox/whatsnew/whatsnew-fx67.html'
         else:
-            template = 'firefox/whatsnew/index.html'
+            if show_default_sync_whatsnew(version) and ftl_file_is_active('firefox/whatsnew/whatsnew-sync'):
+                template = 'firefox/whatsnew/index-sync.html'
+            else:
+                template = 'firefox/whatsnew/index.html'
 
         # return a list to conform with original intention
         return [template]
 
+    def render_to_response(self, context, **response_kwargs):
```

Comment:
This is going to be a much more common pattern I think. I'm going to add (I should have done this before) a `get_ftl_files()` method that you can override to do this kind of thing instead of doing all of this every time.

Reply:
Agree I think this could be a common pattern 👍

Reply:
Updated to use the class dictionary helper